apps/transport/lib/transport_web/live/followed_datasets_live.ex

```diff
@@ -0,0 +1,153 @@
+  use TransportWeb.InputHelpers
+  import Ecto.Query
+  import TransportWeb.DatasetView, only: [icon_type_path: 1]
+  import TransportWeb.Gettext
+  import TransportWeb.InputHelpers
+  import TransportWeb.Router.Helpers
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div class="pb-24">
+      <.form :let={f} for={%{}} phx-change="change" class="search-followed-datasets">
+        <%= search_input(f, :search, value: @search) %>
+        <%= if Enum.count(@select_options) > 1 do %>
+          <%= select(f, :type, [{dgettext("reuser-space", "All"), ""}] ++ @select_options,
+            selected: @type,
+            label: dgettext("reuser-space", "Data type")
+          ) %>
+        <% end %>
+      </.form>
+    </div>
+    <div :if={Enum.empty?(@filtered_datasets)} class="notification">
+      <%= dgettext("reuser-space", "No results") %>
+    </div>
+    <div :if={not Enum.empty?(@filtered_datasets)} class="row">
+      <%= for dataset <- Enum.sort_by(@filtered_datasets, & &1.custom_title) do %>
+        <div class="panel dataset__panel">
+          <div class="panel__content">
+            <div class="dataset__description">
+              <div class="dataset__image">
+                <%= img_tag(DB.Dataset.logo(dataset), alt: dataset.custom_title) %>
+              </div>
+              <div class="dataset__infos">
+                <h3 class="dataset__title">
+                  <a href={dataset_path(@socket, :details, dataset.slug)} target="_blank">
+                    <%= dataset.custom_title %>
+                  </a>
+                </h3>
+                <div class="dataset-localization">
+                  <i class="icon fa fa-map-marker-alt" /><%= DB.Dataset.get_territory_or_nil(dataset) %>
+                </div>
+              </div>
+            </div>
+            <div :if={not is_nil(icon_type_path(dataset))} class="dataset__type">
+              <%= img_tag(icon_type_path(dataset), alt: dataset.type) %>
+            </div>
+          </div>
+          <div class="panel__extra">
+            <div class="dataset__info">
+              <div class="shortlist__notices">
+                <dl class="dataset-format shortlist__notice">
+                  <%= unless dataset |> DB.Dataset.formats() |> Enum.empty?() do %>
+                    <dt class="shortlist__label"><%= dgettext("page-shortlist", "Format") %></dt>
+                    <%= for format <- DB.Dataset.formats(dataset) do %>
+                      <dd class="label"><%= format %></dd>
+                    <% end %>
+                  <% end %>
+                </dl>
+              </div>
+            </div>
+          </div>
+        </div>
+      <% end %>
+    </div>
+    <script type="text/javascript" src={static_path(@socket, "/js/app.js")} />
+    <script nonce={@nonce}>
+      [form] = document.getElementsByClassName("search-followed-datasets");
+      form.onkeydown = function(event) {
+        if (event.key === "Enter") return false;
+      }
```

Comment:
Peut-être serait-il bon d'ajouter du debouncing sur la recherche ?

Reply:
Il y a 300ms [par défaut](https://hexdocs.pm/phoenix_live_view/bindings.html#rate-limiting-events-with-debounce-and-throttle). Un peu plus ou pas ?

Reply:
Ça m'avait échappé. Cool que ça soit built in